Clients/src/presentation/components/Modals/ComplianceFeedback/ComplianceFeedback.tsx

```
@@ -1,18 +1,51 @@
+  "application/zip",
+  "application/x-rar-compressed",
+];
+
 const AuditorFeedback: React.FC<AuditorFeedbackProps> = ({ activeSection }) => {
   const [file, setFile] = useState<File | null>(null);
+  const [error, setError] = useState<string | null>(null);
+  const [isErrorModalOpen, setIsErrorModalOpen] = useState<boolean>(false);
 
   const handleFileUpload = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setError(null);
     if (e.target.files && e.target.files[0]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)